irods_store.py

```diff
@@ -397,7 +412,7 @@ def delete(self, location):
 
```

Comment:
Fixed this as well.

@amit4111989, I didn't mention the log handler in `python-irodsclient`'s `__init__.py` because of [this recent change](https://github.com/irods/python-irodsclient/issues/58).